lib/apiservers/engine/backends/filter/container.go

```diff
@@ -213,14 +214,8 @@ func DockerState(containerState string) string {
 	switch containerState {
 	case "Stopped":
```

Comment:
Given this string is shared between persona and portlayer it should really be pulled from `lib/constants/` package. Not sure how much of a cascade effect this would be.